lib/SILOptimizer/Transforms/CopyPropagation.cpp

```diff
@@ -13,665 +13,28 @@
 /// SSA Copy propagation pass to remove unnecessary copy_value and destroy_value
 /// instructions.
 ///
-/// Because this algorithm rewrites copies and destroys without attempting
-/// to balance the retain count, it is only sound for ownership-SSA.
+/// Because this algorithm rewrites copies and destroys without attempting to
+/// balance the retain count, it is only sound for ownership-SSA. The pass
```

Comment:
`it is only sound for ownership-SSA` -> `it is only sound when SIL is in ownership-SSA form`.